api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/commonAxisSettings/label/template.md

```diff
@@ -5,16 +5,16 @@ default: undefined
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a custom template for the axis label.
 
 ##### param(data): any
-<!-- Description goes here -->
+The label's data object.
```

Comment:
What is this object?

Reply:
In the `markerTemplate`, the data object has a specific type - [BaseChartLegendItem](https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/BaseChartLegendItem/). Yes, the `markerTemplate` article doesn't display this type now, but we're going to fix this. The label template, on the other hand, doesn't specify the type of the `data` parameter.

Reply:
Alexander Polosatov is going to add a type for this parameter